app/assets/javascript/pageflow/linkmap_page/page_type.js

```diff
@@ -431,18 +438,15 @@ pageflow.pageType.register('linkmap_page', _.extend({
   },
 
   isVideoEnabled: function(configuration) {
-    return !pageflow.browser.has('mobile platform') &&
-      (configuration.background_type === 'video' || configuration.background_type === 'hover_video');
+    return configuration.background_type === 'video' || configuration.background_type === 'hover_video';
   },
 
   isBackgroundVideoEnabled: function(configuration) {
-    return !pageflow.browser.has('mobile platform') &&
-      configuration.background_type === 'video';
+    return configuration.background_type === 'video';
   },
```

Comment:
The page type has options to specify mobile fallback images. This option now has no effect anymore. An explicitly configured mobile image should take precedence over the normal background video. Same for hover video.

Reply:
Also the help entry of the page type still says that background videos cannot be played on mobile.